### DISCUSSION TITLE
Contributions not showing on github

### DISCUSSION THREAD

**Discussion by arimtiaz** [community/community#31220] [Other Feature Feedback, Questions, & Ideas]
I have been committing and pushing updates to my own project on Github and it is being updated but the contribution graph is not showing.
<img width="273" alt="image" src="https://user-images.githubusercontent.com/83642378/187187002-1aa85132-41a2-4435-8c18-cc266e82c65b.png">
<img width="1029" alt="image" src="https://user-images.githubusercontent.com/83642378/187187111-a03e2a4d-2360-4458-80aa-4a2c72227d42.png">

**Comment by airtower-luna** [comment]
The commits in that repository aren't recognized as yours. If you [look at the commit list](https://github.com/arimtiaz/ph-ema-john/commits/main), none of them show with your username and avatar.

That usually means the `author.email` on the computer where you made the commits isn't set correctly. GitHub uses the author email address as recorded in the commit to detect which commits are yours, so it must be one of the email addresses on your account. See [Set commit email address](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address) for how to fix it for future commits.

If you want to fix the existing commits you need to [rewrite the repository history](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History).
